IndexedDB/event-dispatch-active-flag.html

```diff
@@ -0,0 +1,153 @@
+<!DOCTYPE html>
+<meta charset=utf-8>
+<title>Transaction active flag is set during event dispatch</title>
+<link rel="help" href="https://w3c.github.io/IndexedDB/#fire-success-event">
+<link rel="help" href="https://w3c.github.io/IndexedDB/#fire-error-event">
+<script src=/resources/testharness.js></script>
+<script src=/resources/testharnessreport.js></script>
+<script src=support.js></script>
+<script>
+
+indexeddb_test(
+  (t, db, tx) => {
+    db.createObjectStore('store');
+  },
+  (t, db) => {
+    const tx = db.transaction('store');
```

Comment:
Can you add a `oncomplete` handler and assert that it hasn't been called on line 38 (where you call finish)? Alternatively, this can all be done in `keep_alive`, but it's less explicit. An assert on line 38 would be a nice and easy way for a reader to make sure that the transaction is still pinned, as expected.